changelog/Microsoft.DirectoryServices.json

```
@@ -26,6 +26,32 @@
     "WorkloadArea": "Identity and access",
     "SubArea": "Directory Management"
   },
+    {
+      "ChangeList": [
+        {
+          "Id": "1c8b1db6-2c25-402f-9bcd-3f5f0fae5a53",
+          "ApiChange": "Relationship",
+          "ChangedApiName": "transitiveRoleAssignments",
+          "ChangeType": "Addition",
+          "Description": "Added the **transitiveRoleAssignments** relationship to the [rbacApplication](https://docs.microsoft.com/en-us/graph/api/resources/rbacApplication?view=graph-rest-beta) resource.",
+          "Target": "rbacApplication"
+        },
+        { 
+          "Id": "1c8b1db6-2c25-402f-9bcd-3f5f0fae5a53",
+          "ApiChange": "Method",
+          "ChangedApiName": "transitiveRoleAssignments",
+          "ChangeType": "Addition",
+          "Description": "Added the [List transitiveRoleAssignments](https://docs.microsoft.com/en-us/graph/api/rbacapplication-list-transitiveroleassignments) method to the [unifiedRoleAssignment](https://docs.microsoft.com/en-us/graph/api/resources/unifiedroleassignment?view=graph-rest-beta) resource.",
+          "Target": "unifiedRoleAssignment"
+        }
+      ],
+        "Id": "1c8b1db6-2c25-402f-9bcd-3f5f0fae5a53",
+        "Cloud": "prd",
+        "Version": "beta",
+        "CreatedDateTime": "2022-03-01T00:00:00.000Z",
```

Change to today's date if you intend to merge today
```suggestion
        "CreatedDateTime": "2022-02-24T00:00:00.000Z",
```